tests/integration/primary_pool_tests.cpp

```diff
@@ -358,7 +358,7 @@ TYPED_TEST(PrimaryPoolTest, coalesce)
   ASSERT_EQ(pool->getBlocksInPool(), 1);
 
   ASSERT_EQ(this->m_allocator->getCurrentSize(), 0);
-  ASSERT_LT(pool->getActualSize(), old_actual_size);
+  ASSERT_LE(pool->getActualSize(), old_actual_size);
```

Comment:
`getActualSize()` and `old_actual_size` now seem to always be equal in the test. I can't judge if this is a meaningful semantic change or not.